joomla-projects/gsoc21_cookie-manager
# Create initial files and folders

## Testing Instructions
* Fetch from the repo
* Check out this branch `initial-folders` 
* Update database structure `System > Database` 
* Discover and install the component


## Initial files and folders created
* com_admin
    * sql
        * updates
            * mysql
                * **4.1.0-2021-06-08.sql**
           * postgresql
                * **4.1.0-2021-06-08.sql**
* com_cookiemanager
    * **access.xml**
    * **config.xml**
    * **cookiemanager.xml**
    * forms
        * **cookie.xml**
        * **filter_cookies.xml**
    * services
        * **provider.php**
    * src
        * Controller
            * **DisplayController.php**
            * **CookieController.php**
            * **CookiesController.php**
        * Model
            * **CookieModel.php**
            * **CookiesModel.php**
        * Table
            * **CookieTable.php**
        * View
            * Cookie
                * **HtmlView.php**
           * Cookies
                * **HtmlView.php**
    * tmpl
        * cookie
            * **edit.php**
        * cookies
            * **default.php**
            * **emptystate.php**
* language
    * en-GB
        * **com_cookiemanager.ini**
        * **com_cookiemanager.sys.ini**
       
* **extensions.sql**
* **base.sql**


![empty](https://user-images.githubusercontent.com/61203226/123204126-14fa4280-d4d5-11eb-9819-9aaad2793935.JPG)

![capture](https://user-images.githubusercontent.com/61203226/123519355-5e5eb380-d6c8-11eb-85d5-64a4e014cbbf.JPG)

![edit-pub](https://user-images.githubusercontent.com/61203226/123048381-f6844080-d41b-11eb-8f09-bc7149a478ee.JPG)

![cat](https://user-images.githubusercontent.com/61203226/123257958-72f84b80-d510-11eb-8592-3854b1f23316.JPG)

![option](https://user-images.githubusercontent.com/61203226/123364830-bf846b00-d592-11eb-8cea-6ff026b40d9e.JPG)
